si-id-conv/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include as_ps/access_rules.as
```

Comment:
wouldn't it be " **si_id_conv**/access_rules.as "?

Reply:
Yes! I copied the whole folder from `pu-conv`. It's been wrong there since ever. Thanks
!

Reply:
I think this might explain an installation issue I noticed a few months back, did not have time to look at at on the spot (beam operation), and forgot about it. nice!